src/mame/drivers/toaplan2.cpp

```diff
@@ -5691,6 +5740,7 @@ GAME( 1994, kingdmgp,    0,        shippumd,   kingdmgp,   toaplan2_state, empty
 GAME( 1994, shippumd,    kingdmgp, shippumd,   shippumd,   toaplan2_state, empty_init,      ROT270, "Raizing / Eighting", "Shippu Mahou Daisakusen (Japan)", MACHINE_SUPPORTS_SAVE )
 
 GAME( 1996, bgaregga,    0,        bgaregga,   bgaregga,   toaplan2_state, init_bgaregga,   ROT270, "Raizing / Eighting", "Battle Garegga (Europe / USA / Japan / Asia) (Sat Feb 3 1996)", MACHINE_SUPPORTS_SAVE )
+GAME( 1996, bgareggap,   bgaregga, bgaregga,   bgaregga,   toaplan2_state, init_bgaregga,   ROT270, "Raizing / Eighting", "Battle Garegga (Prototype) (Wed Jan 17 1996)", MACHINE_SUPPORTS_SAVE )
```

Comment:
This isn’t a prototype, it’s a location test version.  Also, anything that isn’t actually part of the title should only capitalise proper nouns.

Reply:
Understood; there are examples in this file that violate that convention e.g. "Pips & Bibis / Whoopee!!  (Prototype)" or "Ghox (Spinner with .."